.gitignore

```diff
@@ -16,3 +16,1258 @@ __pycache__
 singleheader/simdutf.cpp
 singleheader/simdutf.h
 singleheader/singleheader.zip
+buildclang/cmake_install.cmake
```

Comment:
@Nick-Nuon Can you check this out? I don't think we want to include the buildclang stuff. :-)

Reply:
Yep! Should be fixed!

Slow day today due to doctor's appointment but getting started on vectorized utf8 => Latin1